finality-aleph/src/network/clique/service.rs

```diff
@@ -301,8 +303,8 @@ where
                 },
                 // periodically reporting what we are trying to do
                 _ = status_ticker.tick() => {
-                    info!(target: "validator-network", "Validator Network status: {}", self.manager.status_report());
-                    debug!(target: "validator-network", "Validator Network legacy status: {}", self.legacy_manager.status_report());
+                    info!(target: LOG_TARGET, "Validator Network status: {}", self.manager.status_report());
```

Comment:
So from what I understand, inside clique network, we should know nothing about validators. So shouldn't this and the following status say:
```
Clique Network status: ...
```
Also in this file comments at lines 77 and 98 probably should say clique instead of validator.

Reply:
Yup, good points and hopefully fixed all the remaining instances now, thanks!

Reply:
Just out curiosity, gossip network statuses will also be changed, right? Because right now they say `Network status...` which might be confusing a little. I'm worried I missed that in last review :/

Reply:
This one says `"Clique Network status..."` so it shouldn't be _too_ confusing. Besides they are under different logging targets.